clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -435,7 +436,7 @@ protected void processMessageAndMaybeProduceToKafka(
     switch (msgType) {
       case PUT:
         mergeConflictResult = mergeConflictResolver.put(
-            oldValueProvider,
+            Lazy.of(() -> oldValueProvider.get().getValue()),
```

Comment:
Is this the only call site of `MergeConflictResolver.put()` (outside of tests)? If that is the case, then could we change the signature to take in a `Lazy` containing the new type, rather than using a doubly-wrapped lazy?

Reply:
I started to do that, but it gets kind of hairy fast.  The Lazy<ByteBuffer> trickles into a number of objects in MergeResolver.  ValueAndRmd is where I started to throw my hands up because it uses setters from contexts where it's non trivial to get a version.  So.... I kinda gave up :(

Reply:
Sure, np. Thanks for looking into it!